grapher/stackedCharts/MarimekkoChart.tsx

```diff
@@ -211,13 +212,22 @@ function MarimekkoBarsForOneEntity(props: MarimekkoBarsProps): JSX.Element {
 
     let content = undefined
     if (bars.length) {
-        const allButLast = bars.slice(0, -1)
-        const last = bars[bars.length - 1]
+        const lastNonZeroBarIndex =
+            bars.length -
+            bars.reverse().findIndex((bar) => bar.yPoint.value > 0) -
+            1
+        const predecessors = bars.slice(0, lastNonZeroBarIndex)
+        const lastNonZero = bars.slice(
+            lastNonZeroBarIndex,
+            lastNonZeroBarIndex + 1
+        )
```

Comment:
Why is this not just as below? or is it preferable to use arrays for consistency?
```ts
const lastNonZero = bars[lastNonZeroBarIndex + 1]
```

Reply:
Yeah the idea was to make it more uniform below but it's probably more confusing than is warranted